Yoast/yoast-components
# Implement react hot reloading

## Testing:
* After checking out this branch and installing the new dependencies:
* Run the development environement: `yarn start`
* Make sure you are viewing a component you want to work on in the browser
* Make a (visual) change to that particular component in your IDE and save the file
* If the change shows up without the whole page reloading, `react-hot-loader` is implemented correctly

## Additional information for code reviewer
You can consult the following repos for information about the correct implementation of `react-hot-loader`:
[react-hot-loader](https://github.com/gaearon/react-hot-loader)
[react-hot-loader-boilerplate](https://github.com/gaearon/react-hot-boilerplate/tree/next)
